lib/api/api.go

```diff
@@ -247,6 +247,7 @@ func (s *service) serve(ctx context.Context) {
 	getRestMux.HandleFunc("/rest/db/completion", s.getDBCompletion)              // device folder
 	getRestMux.HandleFunc("/rest/db/file", s.getDBFile)                          // folder file
 	getRestMux.HandleFunc("/rest/db/ignores", s.getDBIgnores)                    // folder
+	getRestMux.HandleFunc("/rest/system/raw-ignore", s.getIgnoresRaw) 		     // folder
```

Comment:
```suggestion
	getRestMux.HandleFunc("/rest/db/ignores/raw", s.getIgnoresRaw) 		     // path
```

Reply:
The ignores strictly speaking don't live in the `db`, but they probably should and we can keep the API endpoint there for the time being... sorry for changing my mind on this.